packages/shared-components/src/components/onto-header/onto-header.tsx

```diff
@@ -1,21 +1,53 @@
-import { Component, Host, h } from '@stencil/core';
+import { Component, Host, h, State } from '@stencil/core';
+import { ServiceProvider, LicenseContextService, License } from '@ontotext/workbench-api'
 
+/**
+ * OntoHeader component for rendering the header of the application.
+ * This component includes a search component, license alert (if applicable),
```

Comment:
Just to point out here, no action is needed. It includes them explicitly, for now. We plan to tweak the plugin system so that we can inject whatever we want declaratively.